### DISCUSSION TITLE
Framer motion slow to update

### DISCUSSION THREAD

**Discussion by matchboxhero** [vercel/next.js#52042] [Help]
Verify canary release

    I verified that the issue exists in the latest Next.js canary release

Provide environment information

Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:19 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T8103
    Binaries:
      Node: 16.14.2
      npm: 9.6.7
      Yarn: 1.22.18
      pnpm: 8.6.5
    Relevant Packages:
      next: 13.4.8-canary.9
      eslint-config-next: 13.4.7
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.1.5
    Next.js Config:
      output: N/A

Which area(s) of Next.js are affected? (leave empty if unsure)

No response
Link to the code that reproduces this issue or a replay of the bug

https://github.com/matchboxhero/next-framer
To Reproduce

Download, install and run the next application
Describe the Bug

The card component on the page should rotate when the mouse is moving, it works but seems really slow to respond and seems to only update properly when the mouse stops moving.

I have lifted the component out of next and into a plain react application and that works as expected, the tutorial I follow does the same thing but with vite and that works as expected too.

This is the working example for referece;
https://shiny3dcard.netlify.app/
https://github.com/arielbk/shiny-3d-card
Expected Behavior

The card should rotate while the mouse is moving in real time
Which browser are you using? (if relevant)

No response
How are you deploying your application? (if relevant)

No response

**Comment by matchboxhero** [comment]
Can anyone help with this, I filed it as a bug as I'm pretty sure it's something to do with nextjs that is causing it to not update in realtime as it should.

Thanks!

**Reply by wave2006** [reply]
same issue,wanner someone can help to give some advise.
